src/components/ui/Drawers/PrimaryBottomDrawer.tsx

```diff
@@ -0,0 +1,101 @@
+'use client';
+
+import { cn } from '@/lib/tailwind/utils';
+
+import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react';
+
+import {
+  BaseDrawerBody,
+  BaseDrawerClose,
+  BaseDrawerContent,
+  BaseDrawerFooter,
+  BaseDrawerHeader,
+  BaseDrawerOverlay,
+  BaseDrawerPortal,
+  BaseDrawerRoot,
+  BaseDrawerTitle,
+  BaseDrawerTrigger,
+} from '.';
+
+const PrimaryBottomDrawerRoot = BaseDrawerRoot;
+const PrimaryBottomDrawerTrigger = BaseDrawerTrigger;
+const PrimaryBottomDrawerClose = BaseDrawerClose;
+const PrimaryBottomDrawerPortal = BaseDrawerPortal;
+const PrimaryBottomDrawerOverlay = BaseDrawerOverlay;
+
+interface IPrimaryBottomDrawerContentProps
+  extends ComponentPropsWithoutRef<typeof BaseDrawerContent> {}
```

Comment:
### [Question & Optional suggestion]
Did you decide not to omit the `side` prop from this `PrimaryBottomDrawer`? 
 Since it's fixed to be used as Bottom drawer, I thought we were going to omit it like this ⏬ 
 ```tsx
 interface IPrimaryBottomDrawerContentProps
  extends Omit<ComponentPropsWithoutRef<typeof BaseDrawerContent>, 'side'> {}
  ```
 With current implementation, I suppose we can still pass `right` to `side` props of `PrimaryBottomDrawer` component.

Reply:
I am sorry I forgot it. Please allow me some time to update it.

Reply:
@kanta1207 

I fixed in this commit: [`cd19587` (#186)](https://github.com/genesis-tech-tribe/nishiki-frontend/pull/186/commits/cd19587537e7345bcfa90a78f44e0ca8af765c5e). Please have a check.